src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -246,6 +246,20 @@
   <data name="AvoidUnsealedAttributesMessage" xml:space="preserve">
     <value>Avoid unsealed attributes</value>
   </data>
+  <data name="AvoidConstArraysTitle" xml:space="preserve">
+    <value>Avoid constant arrays as arguments</value>
+  </data>
+  <data name="AvoidConstArraysCodeFixTitle" xml:space="preserve">
+    <value>Extract to static readonly field</value>
+  </data>
+  <data name="AvoidConstArraysDescription" xml:space="preserve">
+    <value>Having a constant array passed as an argument is not ideally performant. Extract constant arrays as 'static readonly'.</value>
```

Comment:
What about `Constant arrays passed as arguments are not reused which implies a performance overhead. Consider extracting them into 'static readonly' fields to improve performance.`